packit-ci.fmf

```diff
@@ -14,14 +14,14 @@
   prepare:
    - how: shell
      script:
-      - ln -s $(pwd) /var/tmp/rust-keylime_sources
+#      - ln -s $(pwd) /var/tmp/rust-keylime_sources
       - systemctl disable --now dnf-makecache.service || true
       - systemctl disable --now dnf-makecache.timer || true
 
   discover:
     how: fmf
-    url: https://github.com/RedHat-SP-Security/keylime-tests
-    ref: main
+    url: https://github.com/Koncpa/keylime-tests
```

Comment:
I guess that this will be dropped once WIP is removed

Reply:
Yeah..